src/app/pages/properties/properties.component.ts

```diff
@@ -13,8 +14,8 @@ import { Title } from '@angular/platform-browser';
 })
 export class PropertiesComponent implements OnInit {
   properties: Property[] = [];
-  propertiesUrl =
-    'https://landville-backend-web-api.herokuapp.com/api/v1/properties';
+  base_url = APPCONFIG.base_url;
```

Comment:
Kindly take advantage of the environment files that come with Angular to import the base url